src/libs/ErrorMessage/index.js

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import errorMessages from './errorMessages';
 
 /**
- * It's generic function to print any messages.
+ * It's function to print handle error messages in the context of Authenticate.
```

Comment:
Typo? `print handle error messages` is awkward. Probably should be either/or.

Reply:
@deetergp Okay, I think, It will be better. `This function is handling error messages for the Authentication.` else can you advise on it?

Reply:
At the end of the day, shouldn't your naming explain the "what" and your docs and comments explain the "why"? Like shouldn't function be called `showFriendlyErrorMessage` and the doc comment should briefly summarize the reason we need the function like "Display a user-friendly message in the UI when the API returns an authentication error".

Reply:
@deetergp  Okay, Should we change with this `Display a error message in the UI when the API returns an authentication error.`?
If it works, please let me know. I'll change it. Thanks

Reply:
@devkashan1 I meant to move this logic to the context of `Authenticate` (in src/libs/Api.js) since that's the only spot where it's relevant.

Reply:
@yuwenmemon Thanks for the clarification. I just have pushed the changes. You can verify now.